src/fpm/git.f90

```diff
@@ -424,20 +424,22 @@ pure function descriptor_name(descriptor) result(name)
     end function descriptor_name
 
   !> Archive a folder using `git archive`.
-  subroutine git_archive(source, destination, ref, verbose, error)
+  subroutine git_archive(source, destination, ref, additional_files, verbose, error)
```

Comment:
I believe `git archive` is used as a `zip` wrapper such that no further dependencies need be added to the fpm project. So, we can't put `fpm_model.json` here

Reply:
sure, Thanks @perazz .